website/addons/dataverse/views/config.py

```diff
@@ -60,6 +60,11 @@ def dataverse_add_user_account(auth, **kwargs):
         user.external_accounts.append(provider.account)
         user.save()
 
+    user_settings = user.get_addon('dataverse')
```

Comment:
This can be refactored as `user.get_or_add_addon('dataverse')`.

That function returns a value, but if you don't set it to anything it will just be discarded. The effect will be to ensure that the user has the addon enabled. I'd be sure to add a comment! :)